### ISSUE TITLE
[feature] CYOA style pipelines

### ISSUE THREAD

**Issue by Jkwok0714** [TestaDiMucca/Hoarder-Util#77]
## Description

Enables pipelines to link to each other with one entry point, so we run files against a series of tests or rules to determine if they branch to A or B or C etc.

Will require a designed rules system, which relates and should be shared with expanding the filtering modules.

## MVP

Add a rules system that can direct files to A or B.

## Out of scope

If anything is explicitly out of scope - leave empty if not.

**Comment by Jkwok0714** [comment]
Alternatively, be able to target modules in the same pipeline.

**Comment by Jkwok0714** [comment]
Blocked by #140

**Comment by Jkwok0714** [comment]
Branching destinations should also support running another pipeline.
